src/renderer/reader/components/ReaderMenu.tsx

```diff
@@ -469,7 +469,9 @@ const AnnotationCard: React.FC<{ timestamp: number, annotation: IAnnotationState
                     e.preventDefault();
                     const closeNavAnnotation = !dockedMode && !(e.shiftKey && e.altKey);
                     goToLocator(annotation.locatorExtended.locator, closeNavAnnotation);
-                    dispatch(readerLocalActionAnnotations.focus.build(annotation));
+                    
+                    // not used anymore, TODO remove the action
+                    // dispatch(readerLocalActionAnnotations.focus.build(annotation));
```

Comment:
Ah, I think this is still needed, there is no focus steal, just highlighting and "scrolling into view" the currently active annotation. I need to test and verify my assumptions :)